crypto/blake2/build.info

```diff
@@ -0,0 +1,3 @@
+LIBS=../../libcrypto
+SOURCE[../../libcrypto]=\
+        blake2b.c blake2s.c {- $target{blake2b_obj} -}
```

Comment:
Yes, I think we're done for now.  I do want to revisit this in the
not-too-distant future to add SSE support with some asm code, but it looks
good to me as-is.  The triple-license has also been incorporated upstream
now.

On Mon, Feb 29, 2016 at 3:43 PM, Rich Salz <notifications@github.com> wrote:

> In crypto/blake2/build.info
> <https://github.com/openssl/openssl/pull/566#discussion_r54499900>:
>
> > @@ -0,0 +1,3 @@
> > +LIBS=../../libcrypto
> > +SOURCE[../../libcrypto]=\
> > +        blake2b.c blake2s.c {- $target{blake2b_obj} -}
>
> That’s the point of code reviews :) Yes, assembly is harder to integrate.
> But do we think we're done now, from your viewpoint?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openssl/openssl/pull/566/files#r54499900>.
>